config/routes.rb

```diff
@@ -17,7 +17,9 @@
 
     resources :fixed_pages
     resources :links
-    resources :categories
+    resources :categories do
+      resources :orders, module: :categories, only: %i(update), param: :direction, constraints: {direction: /left|right/}
```

Comment:
This route generates the following URL.
```
PUT /editor/categories/:category_id/orders/:direction
```

I think this is not RESTFul because `:direction` is not a resource.

Instead, how about the following URL?
```
PUT /editor/categories/:category_id/order?move_to=:direction
```